itest/starboard_cli_test.go

```diff
@@ -64,6 +64,91 @@ func NewPodSpec(podName string, containers map[string]string, args ...string) (*
 	return pod, err
 }
 
+var _ = Describe("Starboard CLI dryrun", func() {
+
+	Describe("Command init with dryrun", func() {
+
+		It("should return an error if the wrong dryrun flag is provided", func() {
+			out := NewBuffer()
+			err := cmd.Run(versionInfo, []string{
+				"starboard",
+				"init",
+				"--dry",
+				"-v", starboardCLILogLevel,
+			}, out, out)
+			Expect(err).To(HaveOccurred())
+
+			// Expectation: No resources should have been created, so lets check
+
+			crdList, err := customResourceDefinitions.List(context.TODO(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(crdList.Items).Should(HaveLen(0))
+		})
+
+		It("should return an error if dryrun flag provides invalid parameters", func() {
+			out := NewBuffer()
+			err := cmd.Run(versionInfo, []string{
+				"starboard",
+				"init",
+				"--dry-run=client",
+				"-v", starboardCLILogLevel,
+			}, out, out)
+			Expect(err).To(HaveOccurred())
+
+			// Expectation: No resources should have been created, so lets check
+
+			crdList, err := customResourceDefinitions.List(context.TODO(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(crdList.Items).Should(HaveLen(0))
+		})
+
+		It("should show which resources will be created", func() {
+			out := NewBuffer()
```

Comment:
Can we assert on this output as well? See how we check the output for the version command.

Reply:
Done

Reply:
Sure, I have added this now.